jobs/director/templates/director.yml.erb.erb

```diff
@@ -189,6 +189,12 @@ if p('blobstore.provider') == "s3"
    params['blobstore']['options']['s3cli_path'] = "/var/vcap/packages/s3cli/bin/s3cli"
 end
 
+if p('blobstore.provider') == "gcs"
+   params['blobstore']['provider'] = "gcscli"
+   params['blobstore']['options']['gcscli_config_path'] = "/var/vcap/data/tmp/director"
+   params['blobstore']['options']['gcscli_path'] = "/var/vcap/packages/bosh-gcscli/bin/bosh-gcscli"
```

Comment:
Yes, I believe a note in our PR following the style of this [PR](https://github.com/cloudfoundry/bosh/pull/1494) is the way to go.

(By being blocked on merging, I assume you mean into the main cloudfoundry/bosh repo rather than this PR)